### PR TITLE
[5.3] Fix some docblocks

### PR DESCRIPTION
**Pull request by lucasmichot**
Fix some docblocks

### CODE REVIEW

File: src/Illuminate/Mail/Mailable.php [hunk 1]
```diff
@@ -283,8 +283,8 @@ protected function runCallbacks($message)
     /**
      * Set the sender of the message.
      *
-     * @param  object|array|string  $address
-     * @param  string|null  $name
+     * @param  mixed  $address
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Mailable.php]: why mixed?

Comment (lucasmichot) [inline src/Illuminate/Mail/Mailable.php]: This is quite generic enough to be `mixed`

Comment (GrahamCampbell) [inline src/Illuminate/Mail/Mailable.php]: Nope, mixed should be avoided at all costs. It's the same as not documenting it.

File: src/Illuminate/Mail/Mailable.php [hunk 2]
```diff
@@ -100,7 +100,7 @@ class Mailable implements MailableContract
     /**
      * Send the message using the given mailer.
      *
-     * @param  MailerContract  $mailer
+     * @param  \Illuminate\Contracts\Mail\Mailer  $mailer
```
Comment (GrahamCampbell) [inline src/Illuminate/Mail/Mailable.php]: These fixes were still fine tho.

**Pull request closed without merging**